rllib/agents/bandit/bandit_tf_model.py

```diff
@@ -0,0 +1,271 @@
+import gym
+import tensorflow_probability as tfp
+
+from ray.rllib.models.modelv2 import ModelV2
+from ray.rllib.models.tf.tf_modelv2 import TFModelV2
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_tf
+
+tf1, tf, tfv = try_import_tf()
+
+
+class OnlineLinearRegression(tf.Module):
```

Comment:
`tf.Module if tf is not None else object`

(in case tf is not installed)

Reply:
if tf is not installed, I feel like we should just error whenever possible?
we need this module to be tf.Module so the weights of all the arms can be checkpointed actually.